cms-adapter/src/Rsbc.Dmf.CaseManagement.Service/Services/CaseService.cs

```diff
@@ -413,32 +536,36 @@ public async override Task<GetDocumentsReply> GetCaseDocuments(CaseIdRequest req
                         driver.DriverLicenseNumber = item.Driver.DriverLicenseNumber ?? string.Empty;
                         driver.Surname = item.Driver.Surname ?? string.Empty;
                     }
-                    reply.Items.Add(new LegacyDocument
+                    var newDocument = new LegacyDocument
                     {
                         BatchId = item.BatchId ?? string.Empty,
                         DocumentPages = item.DocumentPages,
                         DocumentTypeCode = item.DocumentTypeCode ?? string.Empty,
-                   
                         CaseId = item.CaseId ?? string.Empty,
-                        FaxReceivedDate = Timestamp.FromDateTimeOffset(item.FaxReceivedDate),
-                        ImportDate = Timestamp.FromDateTimeOffset(item.ImportDate),
+                        ImportDate = Timestamp.FromDateTimeOffset(item.ImportDate.Value),
```

Comment:
Because ImportDate is nullable, we can check it for null.  As such we should only set it if there is a value.  Can you move this out of the new statement and put it right after (with a condition to check for null)

Reply:
Fixed all the null checks for the FaxReceivedDate and Import Dates